src/task/models/task.response.model.ts

```diff
@@ -1,5 +1,4 @@
 export interface TaskResponse {
-  _id: string;
```

Comment:
why not return id?

Reply:
The line must have been removed by mistake. I have made the suggested changes. Please check.